cmd/gateway/main.go

```diff
@@ -566,6 +566,15 @@ func startGateway() {
 			}
 		}
 
+		if newConfig.cbCredsAwsId != config.cbCredsAwsId ||
+			newConfig.cbCredsAwsRegion != config.cbCredsAwsRegion ||
+			newConfig.cbCredsAzureId != config.cbCredsAzureId ||
+			newConfig.cbCredsAzureVaultName != config.cbCredsAzureVaultName ||
+			newConfig.cbCredsGcpId != config.cbCredsGcpId ||
+			newConfig.cbCredsGcpProjectId != config.cbCredsGcpProjectId {
+			logger.Warn("config changes for cbCredsAwsId, cbCredsAwsRegion, cbCredsAzureId, cbCredsAzureVaultName, cbCredsGcpId or cbCredsGcpProjectId require a restart")
```

Comment:
Should these warnings use the flag names (cb-creds-aws-id, etc) instead of the field names, since the user will know them by the flag names?

Reply:
Probably.  But would be good to do that to all of them in a separate config.